cinema/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from cinema.views import movie_list, movie_detail
+
+urlpatterns = [
+    path("movies/", movie_list, name="movie-list"),
+    path("movies/<int:pk>", movie_detail, name="movie-detail")
```

Comment:
/ in the end of url
dont forget to read a [checklist.md](https://github.com/mate-academy/py-movie-api/blob/master/checklist.md)